boilerplates/sentry/files/pages/sentry/+Page.tsx

```diff
@@ -0,0 +1,104 @@
+/*{ @if (it.BATI.has("react")) }*/
```

Comment:
Please create 3 new boilerplates `react-sentry`, `vue-sentry` and `solid-sentry` for this purpose (With the right `flag` combination in package.json).

This file should then be split and moved to its respective boilerplates.